src/test/fixtures/fixtures_test.ts

```diff
@@ -142,7 +140,7 @@ suite('Fixtures', () => {
         // 2. Compare the generated output to the expected conversion.
         //    Output the diff & fail if any differences are encountered.
         const diffResult = dircompare.compareSync(
-            fixtureResultDir, fixtureExpectedDir, {compareSize: true});
+            fixtureResultDir, fixtureExpectedDir, {compareSize: false});
```

Comment:
why can't we compare sizes?

Reply:
I don't remember exactly, it was one of the things i disabled initially. I will double check and try to get you a solid reason.

Reply:
![image](https://user-images.githubusercontent.com/855184/35171146-502a2c72-fd28-11e7-8a00-8d7c9ffec1fc.png)

Looking into this more.

Reply:
Looks like generated is generating `\r\n` and expected is `\n`.